content/applications/inventory_and_mrp/manufacturing/workflows/subcontracting_basic.rst

```diff
@@ -0,0 +1,200 @@
+
+Configure BoM
+-------------
+
+To configure a |BoM| for basic subcontracting, click the :guilabel:`Bill of Materials` smart button
+on the product's page, and select the |BoM|.
+
+Alternatively, navigate to :menuselection:`Manufacturing app --> Products --> Bills of Materials`,
+and select the |BoM| for the subcontracted product.
+
+.. seealso::
+   For a full overview of |BoM| configuration, see the :doc:`Bill of materials
+   <../basic_setup/bill_configuration>` documentation.
+
+In the :guilabel:`BoM Type` field, select the :guilabel:`Subcontracting` option. Then, add one or
+more subcontractors in the :guilabel:`Subcontractors` field that appears below.
+
+.. image:: subcontracting_basic/bom-type.png
+   :align: center
+   :alt: The "BoM Type" field on a BoM, configured to manufacture the product using subcontracting.
+
+Finally, click on the :guilabel:`Miscellaneous` tab. In the :guilabel:`Manuf. Lead Time` field,
+enter the number of days it takes the subcontractor to manufacture the product. This number is
+factored in when calculating the product's expected arrival date.
+
+.. note::
+   When using basic subcontracting, there is no need to list components in the :guilabel:`Components`
+   tab of the |BoM|, since the components required for manufacturing, and the means by which they
+   are acquired, are handled by the subcontractor.
+
+Basic subcontracting workflow
+=============================
+
+The basic subcontracting workflow consists of up to four steps:
+
+#. Create a sales order (SO) for the subcontracted product; doing so creates a |PO| to purchase the
+   product from the subcontractor.
+#. Confirm the |PO| created in the previous step, or create a new |PO|; doing so creates a receipt
+   order or a dropship order.
+#. Process the receipt once the subcontractor has finished manufacturing the subcontracted product,
+   and shipped it back to the contracting company, **OR** process the dropship order to ship the
+   product directly to the customer.
+#. If the workflow was started by creating an |SO|, and the finished product is not dropshipped to
+   the end customer, process the delivery order once the product is shipped to the customer.
+
+The specific number of steps depends on the reason that the subcontracted product is being purchased
+from the subcontractor.
+
+If the reason is to fulfill a specific customer order, the process starts with creating an |SO|, and
+ends with delivering the product to the customer, or having the subcontractor dropship it to them.
+
+If the reason is to increase the quantity of stock on-hand, the process starts with creating a |PO|,
+and ends with receiving the product into inventory.
+
+Create SO
+---------
+
+It is only necessary to complete this step if the product is being purchased from the subcontractor
+to fulfill a customer need. If the product is being purchased to increase the quantity of stock
+on-hand, move on to the next step.
```

Comment:
could be beneficial to add a link/anchor to the next step, so they can quickly click it and be directed to the appropriate section.